src/libs/antares/series/series.cpp

```diff
@@ -74,24 +72,17 @@ int TimeSeries::saveToFile(const std::string& filename, bool saveEvenIfAllZero)
 
 double TimeSeries::getCoefficient(uint32_t year, uint32_t timestep) const
 {
-    if (timeSeries.width == 0)
```

Comment:
**Explanation** : by construction, **timeSeries** can never have a width == 0, so we don't need the previous complexity